composer.json

```diff
@@ -12,6 +12,14 @@
     "authors": [
         {
             "name": "Raphael Schweikert"
+        },
+        {
+            "name": "Oliver Klee",
+            "email": "github@oliverklee.de"
+        },
+        {
+            "name": "Jake Hotson",
+            "email": "jake@qzdesign.co.uk"
```

Comment:
Could you please use `jake.github@qzdesign.co.uk` here?  It helps reduce spam (or at least let me know where it came from).

Reply:
Done.